src/lib/core/command.ts

```diff
@@ -0,0 +1,23 @@
+import * as rx from "rxjs";
```

Comment:
So in general it's better to import every operator you want to use manually (also for tree-shaking capability).

We're only using `fromEvent` here, so we don't need to import the entire rxjs library. Rx distinguishes between creational operators and chainable operators. Creational ones can be imported straight from `rxjs`. Chainable operators are imported from `rxjs/operators`.

So in this case we want to do

```
import { fromEvent } from 'rxjs';
```

instead.